src/other/broadcasting.jl

```diff
@@ -225,6 +227,8 @@ function Base.Broadcast.broadcast_unalias(dest::AbstractDataFrame, src)
     return src
 end
 
+Base.Broadcast.broadcast_unalias(::Nothing, src::AbstractDataFrame) = src
```

Comment:
I think this is normally never called AFAICT but we want to avoid introducing dispatch ambiguities in broadcasting.